api/src/opentrons/protocol_engine/types.py

```diff
@@ -51,9 +51,10 @@ class ModuleLocation(BaseModel):
     )
 
 
-OffDeckLocationType = Literal["off-deck"]
+_OffDeckLocationType = Literal["off-deck"]
+OFF_DECK_LOCATION: _OffDeckLocationType = "off-deck"
```

Comment:
We [wanted](https://github.com/Opentrons/opentrons/pull/11521#discussion_r987980444) to change this to camelCase, right?

```suggestion
_OffDeckLocationType = Literal["offDeck"]
OFF_DECK_LOCATION: _OffDeckLocationType = "offDeck"
```

Reply:
Woops